ui-charts/src/manchette/hooks/useManchetteWithSpaceTimeChart.ts

```diff
@@ -58,6 +70,14 @@ type State = {
   scales: SpaceScale[];
 };
 
+export type SplitPoint<Payload = unknown> = {
+  position: number;
+  size: number;
+  payload: Payload;
+  spaceTimeChartComponent: ComponentType<{ position: number; size: number; payload: Payload }>;
+  manchetteComponent: ComponentType<{ position: number; size: number; payload: Payload }>;
```

Comment:
What's the point in passing a payload and components, instead of two React nodes?

Reply:
My main beef with the node version was that it was using callbacks. Usually props contain already-built elements (e.g. `children`). How would this look?

```ts
export type SplitPoint = {
  position: number;
  size: number;
  spaceTimeChartElement: ReactNode;
  manchetteElement: ReactNode;
};
```

Reply:
But, maybe `spaceTimeChartElement` is unnecessary here, maybe it can be dropped and left for the user to pass to the space time chart on their own.

Reply:
I don't see any issue with the `ReactNode` version, I think it is better indeed. About the `spaceTimeChartElement` though, I like that everything split-points related is in the same spot. It makes it way easier to manage, I think.

Reply:
I just pushed a fix for this 👍